docs/Configuration/Miscellaneous/Hangup-Cause.md

```diff
@@ -46,63 +46,59 @@ Note that in some cases, the hangup causes returned may not be reflected in . Fo
 #### Example
 
 This example illustrates obtaining hangup cause information for a parallel dial to *SIP/foo* and *SIP/bar*. A hangup handler has been attached to the calling channel, which executes the subroutine at **handler,s,1** when the channel is hung up. This queries the hangup cause information using the HANGUPCAUSE\_KEYS function and the HANGUPCAUSE function. The channels returned from HANGUPCAUSE\_KEYS are parsed out, and each is queried for their hangup cause information. The technology specific cause code as well as the Asterisk cause code are printed to the CLI.
-
+```
 [default]
-
 exten => s,1,NoOp()
- same => n,Set(CHANNEL(hangup\_handler\_push)=handler,s,1)
- same => n,Dial(SIP/foo&SIP/bar,10)
- same => n,Hangup()
+same => n,Set(CHANNEL(hangup_handler_push)=handler,s,1)
+same => n,Dial(SIP/foo&SIP/bar,10)
+same => n,Hangup()
```

Comment:
Thanks for fixing the code blocks and the backslashes.

Actually though, we tend to indent the `same` lines by a space so the `=>` line up...

```
exten => s,1,NoOp()
 same => n,Set(CHANNEL(hangup_handler_push)=handler,s,1)
 same => n,Dial(SIP/foo&SIP/bar,10)
 same => n,Hangup()
```

Reply:
sames are now lined up.